.github/workflows/continuous-integration.yml

```diff
@@ -485,8 +559,8 @@ jobs:
   testing-reports:
     name: Testing Reports
     runs-on: ubuntu-latest
-    needs: cypress-tests
-    if: ${{ always() && (needs.cypress-tests.result == 'success' || needs.cypress-tests.result == 'failure') }}
+    needs: [cypress-tests-prep, cypress-tests]
```

Comment:
We need `cypress-tests-prep` because of the `if` statement below it:

```
if: ${{ always() && needs.cypress-tests-prep.outputs.num_containers > 0 && (needs.cypress-tests.result == 'success' || needs.cypress-tests.result == 'failure') }}
```